images/inference-services/seldon-core/run.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+set -e
+set -u
+set -o pipefail
+
+# load org and project from repository if exists,
+# if not, set them as a random string
+if [ -e ".fuseml/_project" ]; then
+    export ORG=$(cat .fuseml/_org)
+    export PROJECT=$(cat .fuseml/_project)
+else
+    export ORG=$(tr -dc a-z0-9 </dev/urandom | head -c 6 ; echo '')
+    export PROJECT=$(tr -dc a-z0-9 </dev/urandom | head -c 6 ; echo '')
+fi
+
+export S3_ENDPOINT=${MLFLOW_S3_ENDPOINT_URL/*:\/\//}
+
+mc alias set minio ${MLFLOW_S3_ENDPOINT_URL} ${AWS_ACCESS_KEY_ID} ${AWS_SECRET_ACCESS_KEY}
+model_bucket="minio${FUSEML_MODEL//s3:\//}"
+
+export PROTOCOL_VERSION="v1"
```

Comment:
`PROTOCOL_VERSION` is not being used anywhere.

Reply:
ah. true, copy&paste ...